husky_directory/templates/search.html

```diff
@@ -6,7 +6,7 @@ <h3>Search by</h3>
                             <div class="input-group">
                                 <input type="text" class="form-control"
                                        {% if search_result is defined %}
-                                           value="{{ search_result['query'].values()|first }}"
+                                           value="{{ search_result['request'].values()|first }}"
```

Comment:
[One more](https://github.com/UWIT-IAM/uw-husky-directory/blob/main/husky_directory/templates/search.html#L14) changed needed in this file

Reply:
I disagree. I think at that state, it is the user's query that they are inputting. It is not yet a request. It's the search field for their query, that we manipulate.

Reply:
When we return a response, we include the request we received.

Reply:
(I purposely did not change that one.)

Reply:
To put it more concisely:

From the context of the front-end, the user is entering a query.
From the context of the back-end, the server is including the original request with the response.

Reply:
Ok, this should still work with the [js](https://github.com/UWIT-IAM/uw-husky-directory/blob/main/husky_directory/static/js/setSearchNameAttribute.js#L10).

Reply:
I'll give it a go.

Reply:
yes, i mentioned i tested this...

Reply:
(don't forget, you can too!)